src/client/components/Post.jsx

```diff
@@ -1,44 +1,92 @@
 import { useState, useEffect } from "react";
 import "../styling/post.css"
 
+import PostComment from "./PostComment"
+
+const postId = 1
+
 const URL = process.env.REACT_APP_API_URL;
-const postEndpoint = "/post/1"
-const commentEndpoint = "/comment/1"
+const postEndpoint = `/post/${postId}`
 const postURL = URL + postEndpoint
-const commentURL = URL + commentEndpoint
 
-const Post = () => {
+const userDummyData = {
+    id: 1,
+    username: "NathanTheSamosa",
+    password: "my-secret",
+    email: "nathanthesamosa@gmail.com",
+    admin: false
+}
 
-    const userDummyData = {
-        id: 1,
-        username: "NathanTheSamosa",
-        password: "my-secret",
-        email: "nathanthesamosa@gmail.com",
-        admin: false
-    }
+const Post = () => {
 
     const [postData, setPostData] = useState(null)
-    const [commentArr, setCommentArr] = useState([])
+    const [commentArr, setCommentArr] = useState()
```

Comment:
How come you removed the `[]` ?

Reply:
good catch, we `setCommentArr(construction)` where construction is an object.... should have renamed commentArr to imply it was an object rather than an array!

Reply:
Hmmm - not sure about the naming here - it's never a good idea to append `Arr`, or `Obj` etc. Better is to name the variable appropriately as to what it represent. So maybe `comments`, here? And what does `construction` mean? What does it represent?

Reply:
Why is it not a good idea to describe a variable based on its type? In my mind, that is the best way to describe what it represents.

Construction represents the tree of comments constructed from the fetched raw data. Its important it has a name seperate from something like `comment` as the recursive functions are much more readable with a "level zero" that everything builds up from.

Reply:
I guess don't see how `comments` is more descriptive or explicit of a name than `commentArr`, why is including the variable's type a bad idea?